nova/virt/vmwareapi/vmops.py

```diff
@@ -1535,6 +1533,18 @@ def _destroy_instance(self, context, instance, destroy_disks=True):
                 self._session._call_method(self._session.vim,
                                            "UnregisterVM", vm_ref)
                 LOG.debug("Unregistered the VM", instance=instance)
+                if vm_group:
+                    cluster_config = self._session._call_method(
+                        vutil, "get_object_property", self._cluster,
+                        "configurationEx")
+                    for group in cluster_config.group:
+                        if group.name == vm_group.name:
+                            if not hasattr(group, 'vm'):
```

Comment:
Are you sure that the group will not have the attribute anymore and that it's not an empty list?

Reply:
Yes - double checked it. when the last vm is removed is not [] but removed property instead.

Reply:
Ok. Now it's triple-checked :smile: